VoxelSniperForge/src/main/java/com/github/kevindagame/voxelsniperforge/VoxelSniperForge.java

```diff
@@ -243,4 +244,14 @@ public static boolean isTreeType(@NotNull ConfiguredFeature<?, ?> feature) {
                 feature.config() instanceof HugeFungusConfiguration ||
                 feature.feature() instanceof ChorusPlantFeature;
     }
+
+    @NotNull
+    public ForgeWorld getWorld(@NotNull ServerLevel level) {
+        var str = level.toString();
+        var name = str.substring("ServerLevel[".length(), str.length() - 1);
```

Comment:
Why aren't you using the full name here?

Reply:
Good point, I guess that way it was exactly the same as the spigot version, but that isn't necessary anyway, changed it now